### DISCUSSION TITLE
Vaultwarden refuses to connect over specified port

### DISCUSSION THREAD

**Discussion by Slips-PC** [dani-garcia/vaultwarden#2554] [Q&A] [answered]
### Subject of the issue
When I try to connect to vaultwarden in any way, all connections are refused.

### Deployment environment

* vaultwarden version: 1.25.0

* Install method: Docker Image

* Clients used: Web, extension

* Reverse proxy and version: nginx/1.18.0

* MySQL/MariaDB or PostgreSQL version: 13+225

* Other relevant details:

### Steps to reproduce
 The vaultwarden docker image was started using the command `docker run -d --name vaultwarden -v /vw-data/:/data/ -p 8185:8185 vaultwarden/server:latest` and connection was attempted both over a reverse proxy and directly connecting to the IP.

### Expected behaviour
The web vault is shown.

### Actual behaviour
Vaultwarden's connection fails, either via a 502 for nginx or a refused connection when attempting to connect over the port.

### Troubleshooting data

The vaultwarden docker logs don't suggest anything to be out of the ordinary.
```
root@slipfox:~# docker logs vaultwarden
/--------------------------------------------------------------------\
|                        Starting Vaultwarden                        |
|                           Version 1.25.0                           |
|--------------------------------------------------------------------|
| This is an *unofficial* Bitwarden implementation, DO NOT use the   |
| official channels to report bugs/features, regardless of client.   |
| Send usage/configuration questions or feature requests to:         |
|   https://vaultwarden.discourse.group/                             |
| Report suspected bugs/issues in the software itself at:            |
|   https://github.com/dani-garcia/vaultwarden/issues/new            |
\--------------------------------------------------------------------/

[INFO] No .env file found.

[DEPRECATED]: `SMTP_SSL` or `SMTP_EXPLICIT_TLS` is set. Please use `SMTP_SECURITY` instead.
[2022-06-10 18:40:16.245][start][INFO] Rocket has launched from http://0.0.0.0:80
```
It was working previously, but stopped sometime in the past day.

**Comment by Masgalor** [comment, marked as answer]
Vaultwarden is listening on port 80 `Rocket has launched from http://0.0.0.0:80` but you are bridging port 8185 `-p 8185:8185 `.

I am not that fit with docker but I guess you should use `-p 8185:80 `.

**Reply by Slips-PC** [reply]
Hm, that worked. Thank you.
